components/canvas/lib.rs

```diff
@@ -22,3 +26,13 @@ pub mod canvas_paint_thread;
 mod webgl_limits;
 mod webgl_mode;
 pub mod webgl_thread;
+
+pub enum ConstellationCanvasMsg {
```

Comment:
If we put this message type into `canvas_traits` could we make constellation not depend on `canvas`?

Reply:
Right now the constellation is responsible for starting the canvas paint thread. I can file a followup to investigate removing that, but I'd rather not delay this work for it.